plugins/modules/eos_ospfv2.py

```diff
@@ -0,0 +1,1458 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for eos_ospfv2
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "network",
+}
+
+DOCUMENTATION = """
+---
+module: eos_ospfv2
+version_added: 2.10
+short_description: Configures and manages attributes of ospf on Arista EOS platforms.
+description: This module configures and manages the attributes of ospf on Arista EOS platforms.
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+options:
+  config:
+    description: A list of configurations for ospf.
+    type: list
+    elements: dict
+    suboptions:
+      processes:
+        description: A list of dictionary specifying the ospf processes.
+        type: list
+        elements: dict
+        suboptions:
+          process_id:
+            description: ID of OSPFV2 process.
+            type: int
+            required: True
+          vrf:
+            description: VRF name .
+            type: str
+          traffic_engineering:
+            description: Enter traffic engineering config mode
+            type: bool
+          adjacency:
```

Comment:
`running_config` option needs to be added to doc string. Along with the command whose output is required:
Example:
```
description:
    - The module, by default, will connect to the remote device and retrieve the current
      running-config to use as a base for comparing against the contents of source.
      There are times when it is not desirable to have the task get the current running-config
      for every task in a playbook.  The I(running_config) argument allows the implementer
      to pass in the configuration to use as the base config for comparison. This
      value of this option should be the output received from device by executing
      command B(show running-config router static).
```

Reply:
**UPDATE:** 

The description we use for running_config is not correct at the moment. It is similar to what we have in https://docs.ansible.com/ansible/latest/modules/ios_config_module.html#parameter-running_config. But the RM's running_config is different from the *_config one. What do you think about updating it to - https://github.com/ansible-collections/cisco.nxos/pull/33/files#diff-8f4ac16689eeae1ccce530dea6d10c9bR51?